libcontainer/utils/utils.go

```diff
@@ -37,53 +35,6 @@ func WriteJSON(w io.Writer, v any) error {
 	return err
 }
 
-// CleanPath makes a path safe for use with filepath.Join. This is done by not
-// only cleaning the path, but also (if the path is relative) adding a leading
-// '/' and cleaning it (then removing the leading '/'). This ensures that a
-// path resulting from prepending another path will always resolve to lexically
-// be a subdirectory of the prefixed path. This is all done lexically, so paths
-// that include symlinks won't be safe as a result of using CleanPath.
-func CleanPath(path string) string {
```

Comment:
Does this count as a breaking change? Maybe we should add a deprecation warning in `v1.4.0` and remove it in `v1.5.0`.